src/services/sessionInvite.js

```diff
@@ -738,54 +833,61 @@ module.exports = class UserInviteHelper {
 
 			const OutputCSVData = []
 			modifiedCsv.forEach((row) => {
-				const {
-					title,
-					description,
-					recommended_for,
-					categories,
-					medium,
-					time_zone,
-					mentor_id,
-					mentees,
-					type,
-					date,
-					time24hrs,
-					duration,
-					status,
-					statusMessage,
-					action,
-					id,
-					meeting_info,
-				} = row
+				const { meeting_info, status, statusMessage, ...restOfRow } = row // Destructure meeting_info, status, and statusMessage separately
+				const mappedRow = {}
+
+				Object.keys(restOfRow).forEach((key) => {
+					let mappedKey = key // Default to the original key
+
+					// Custom mapping for specific fields
+					switch (key) {
+						case 'mentor_id':
+							mappedKey = 'Mentor(Email)'
+							mappedRow[mappedKey] = restOfRow[key]
+							break
+						case 'mentees':
+							mappedKey = 'Mentees(Email)'
+							mappedRow[mappedKey] = restOfRow[key].join(', ')
+							break
+						case 'date':
+							mappedKey = 'Date(DD-MM-YYYY)'
+							mappedRow[mappedKey] = restOfRow[key]
+							break
+						case 'time_zone':
+							mappedKey = 'Time Zone(IST/UTC)'
+							mappedRow[mappedKey] = restOfRow[key]
+							break
+						case 'time24hrs':
+							mappedKey = 'Time (24 hrs)'
+							mappedRow[mappedKey] = restOfRow[key]
+							break
+						case 'duration':
+							mappedKey = 'Duration(Min)'
+							mappedRow[mappedKey] = restOfRow[key]
+							break
+						default:
+							mappedRow[key] = restOfRow[key] // Use the original key
+					}
```

Comment:
@rakeshSgr , It will be mapped based on existing name , because entity_types will be given how it is there in the db and it won't accept if the entoties are not according to the entity labels